packages/tomls/src/oracle_adapters/configs/init_lm_token_prices.toml

```diff
@@ -1,11 +1,41 @@
+[invoke.oracle_adapters_reya_lm_selini_init_price_range]
+target = ["OracleAdaptersProxy"]
+fromCall.func = "owner"
+func = "setLmTokenPriceConfiguration"
+args = [
+    "<%= settings.reyaLmAssetPairIdSELINI %>",
+    { priceLowerBound = "<%= parseEther('0.9') %>", priceUpperBound = "<%= parseEther('2') %>" },
+]
+depends = ["invoke.oracle_adapters_reya_lm_price_updater_owner"]
+
+[invoke.oracle_adapters_reya_lm_amber_init_price_range]
+target = ["OracleAdaptersProxy"]
+fromCall.func = "owner"
+func = "setLmTokenPriceConfiguration"
+args = [
+    "<%= settings.reyaLmAssetPairIdAMBER %>",
+    { priceLowerBound = "<%= parseEther('0.9') %>", priceUpperBound = "<%= parseEther('2') %>" },
+]
+depends = ["invoke.oracle_adapters_reya_lm_price_updater_owner"]
+
+[invoke.oracle_adapters_reya_lm_hedge_init_price_range]
+target = ["OracleAdaptersProxy"]
+fromCall.func = "owner"
+func = "setLmTokenPriceConfiguration"
+args = [
+    "<%= settings.reyaLmAssetPairIdHEDGE %>",
+    { priceLowerBound = "<%= parseEther('0.5') %>", priceUpperBound = "<%= parseEther('1.5') %>" },
+]
+depends = ["invoke.oracle_adapters_reya_lm_price_updater_owner"]
+
 [invoke.oracle_adapters_reya_lm_selini_init_price]
 target = ["OracleAdaptersProxy"]
 fromCall.func = "owner"
 func = "updateLmTokenPrice"
 args = [
     { assetPairId = "<%= settings.reyaLmAssetPairIdSELINI %>", price = "<%= parseEther('1') %>", vestingPeriodInSeconds = "0" },
 ]
-depends = ["invoke.oracle_adapters_reya_lm_price_updater_owner"]
+depends = ["invoke.oracle_adapters_reya_lm_selini_init_price_range"]
```

Comment:
is this going to accidentally cause the selini token price to get updated back to 1 when we execute it?

Reply:
I don't think it should but let's be double check at simulation

Reply:
commented them out such that there's no way of thins happening, and I've also bounded the share price to be 1.03-1.05 for rAmber and rSelini.